scripts/smoke/check.js

```diff
@@ -6,9 +6,11 @@ import * as path from 'node:path';
 import pLimit from 'p-limit';
 import { tsconfigResolverSync } from 'tsconfig-resolver';
 
+const skippedExamples = ['toolbar-app', 'component']
```

Comment:
`astro check` is able to handle those examples, because at the end of the day it's kinda just `tsc` + Astro support, but it's kinda not the purpose, so figured it's safe to skip them.